src/sample/RedeemAccessTokenSample.js

```diff
@@ -13,6 +13,7 @@ class RedeemAccessTokenSample {
      * @return {Object} balance0 - balance of one account
      */
     static async use(grantee, tokenId) {
+        // forAccessToken snippet begin
```

Comment:
These pair of comments will serve as document sample code anchor, so that I can change the sample code in between. If I change the sample code directly it will break the documentation.